app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class IngredientsController < ApplicationController
+
+  def index 
+    @ingredients = Ingredient.all.order(:name) 
```

Comment:
The AR for `.order` is correct, but since it's changing the collection in a dynamic way, it should be tested as a model method - remember that the models are responsible for data calculations & querying the Db directly, not the controller, so this would be an MVC infraction.